.circleci/config.yml

```diff
@@ -39,7 +39,7 @@ jobs:
           name: Install CircleCI-specific dependencies
           command: |
             sudo apt update
-            sudo apt install libenchant1c2a
```

Comment:
Is it not needed by pylint for spell-checking anymore ?

Reply:
_My bad_.